static/js/services/vulnerability-service.js

```diff
@@ -502,14 +504,14 @@ angular.module('quay').factory('VulnerabilityService', ['Config', 'ApiService',
       'description': 'This metric measures the impact on confidentiality of a successfully exploited vulnerability. Confidentiality refers to limiting information access and disclosure to only authorized users, as well as preventing access by, or disclosure to, unauthorized ones. Increased confidentiality impact increases the vulnerability score.',
       'values': [
         {
-          'id': 'C',
-          'title': 'Complete',
+          'id': 'H',
```

Comment:
Is there a document that describes these updated values that you could include in the PR?

Reply:
https://www.first.org/cvss/v3.1/specification-document#Vector-String
As suggested by @hdonnay